### PR TITLE
Endwalker update

### PR DESCRIPTION
**Pull request by Dozer1170**
Get plugin working in Endwalker, tweak settings for dragoon, add a priority tab to set action priority.

### CODE REVIEW

**Comment by tobilen** [comment]
can confirm this is working fine for me. it does not have sage or reaper actions as selectable reminders. the priority feature needs some documentation on how to use it though.

thanks for your work 👍

**Comment by Dozer1170** [comment]
@tobilen i unfortunately know nothing about those jobs so perhaps someone with more knowledge would be better suited to adding their actions. I will add some documentation or instructions for priority soon. Thanks for testing!

**Comment by tobilen** [comment]
> @tobilen i unfortunately know nothing about those jobs so perhaps someone with more knowledge would be better suited to adding their actions. I will add some documentation or instructions for priority soon. Thanks for testing!

if you point me to where the action lists are maintained, i can see about creating a PR onto this one for the sage stuff at least

**Comment by Dozer1170** [comment]
I have not had to add any actions so I am not sure where you would do so.

**Comment by ercarp** [comment]
How do I install this?

**Comment by tobilen** [comment]
@ercarp

**Easy but Unsafe***
- Download and extract this zip into  `%APPDATA%\XIVLauncher\installedPlugins`
[RemindMe.zip](https://github.com/Caraxi/RemindMe/files/7798780/RemindMe.zip)
- You might have to delete any folders other than `1.2.1.0` in `%APPDATA%\XIVLauncher\installedPlugins\RemindMe`.


**Complicated but Safe (for people who know c# / are coders):**
- Check out the fork, switch to the branch `endwalker_update`, install Visual Studio and compile project
- Move .dll and .pdb files into `%APPDATA%\XIVLauncher\installedPlugins\RemindMe\1.x.x.x` (replace x with whatever version you have installed)

*the easy method is unsafe because you need to trust the .dll file in the zip archive, which i compiled on my machine.

**Comment by elvenmad** [comment]
Latest update 1.2.0.0 works fine just like pre Endwalker as long as you do not play Sage or Reaper, nothing works for those jobs not even the cross class ability's all RemindMe buffs,skills and ability windows are blank for Sage and Reaper even if you set up cross job/role abiltys up on another job as soon as you spec Sage or Reaper there gone

**Comment by tobilen** [comment]
i updated my comment with a compiled version that includes RPR and SGE, based on the fix from @imynwulf (https://github.com/Caraxi/RemindMe/issues/57#issuecomment-1003636858)

**Comment by nyxalteria** [comment]
issues ive encountered within 1h of testing

ast:
-astro draw does not track correctly, only shows  up with 2 charges, no CD
-celestial intersection has same as above, possibly due to charge system
-essential dignity is having the same issue due to the charge system

sage:
-when tracking all 3 phlegmas (due to sync purposes) it shows all 3, so it is not recognizing them correctly at max level, it recognizes them correctly when synced, but not at 90 with all 3 selected (its skill upgrade 1>2>3 via traits)
-the dot is not trackable AT ALL regardless of level, possibly due to the way the skill is coded (needing to use eukrasia first)

whm:
-divine benison does not track due to charge upgrade via trait (same problem as astro stuff)

rdm:
-acceleration has the same problem to due charges
-corps a corps has same problem due to charges

basically anything with charges is borked and does not work unless thats how its suppose to work(unsure, a lot of these classes did not have things with charges prior to EW), but thats not helpful for tracking things with charges you want to use on cd.

**Comment by tobilen** [comment]
i think the problem with charges existed before, although i'm not sure. definitely think its something that needs to be addressed, though maybe not in this PR. 👍 

> -when tracking all 3 phlegmas (due to sync purposes) it shows all 3, so it is not recognizing them correctly at max level, it recognizes them correctly when synced, but not at 90 with all 3 selected (its skill upgrade 1>2>3 via traits)

did you try only tracking base `phlegma`? since its available whenever the upgrades are available, it should work fine, no?

**Comment by nyxalteria** [comment]
> i think the problem with charges existed before, although i'm not sure. definitely think its something that needs to be addressed, though maybe not in this PR. 👍
> 
> > -when tracking all 3 phlegmas (due to sync purposes) it shows all 3, so it is not recognizing them correctly at max level, it recognizes them correctly when synced, but not at 90 with all 3 selected (its skill upgrade 1>2>3 via traits)
> 
> did you try only tracking base `phlegma`? since its available whenever the upgrades are available, it should work fine, no?



can confirm that doesn't work lol, no charges available, shows as available (this is an alt with phlegma 2 since its only at 78 but its generally the same issue)

image

**Comment by tobilen** [comment]
@Dozer1170 i've created a PR on top of your PR that adresses most of the issues: https://github.com/Dozer1170/RemindMe/pull/1

**Comment by Dozer1170** [comment]
> @Dozer1170 i've created a PR on top of your PR that adresses most of the issues: https://github.com/Dozer1170/RemindMe/pull/1

Awesome! Thanks so much

**Comment by nyxalteria** [comment]
> @Dozer1170 i've created a PR on top of your PR that adresses most of the issues: [Dozer1170#1](https://github.com/Dozer1170/RemindMe/pull/1)

Added the newly compiled version and still not seeing options to track sge DOT. Phlegma is now having the issue that the other above skills were where its not showing unless it has both charges. (I also checked "show gcd" and theres nothing about eukrasian dosis 1-2)

image

**Comment by tobilen** [comment]
> > @Dozer1170 i've created a PR on top of your PR that adresses most of the issues: [Dozer1170#1](https://github.com/Dozer1170/RemindMe/pull/1)
> 
> Added the newly compiled version and still not seeing options to track sge DOT. Phlegma is now having the issue that the other above skills were where its not showing unless it has both charges. (I also checked "show gcd" and theres nothing about eukrasian dosis 1-2)
> 
> image

mh, sounds like there is something wrong with your installation. You definitely should have SGE status effects there. Do you have only one numbered folder under the`%APPDATA%\XIVLauncher\installedPlugins\RemindMe` directory? 🤔 

And the charges is a separate issue that i did not touch. Could this be because it gets upgraded to two charges as you level up?

**Comment by nyxalteria** [comment]
fixed it, something else was wrong o7

> Could this be because it gets upgraded to two charges as you level up?

don't believe so, for some of the skills like astros draw you have 2 charges by default and it has the same issue, its just a charge system tracking issue i think, because the skills that formerly didn't have charges use to work fine, astro for instance made draw have 2 charges by default and some classes gain charges on level up from traits, but they both have the same issue

**Comment by nyxalteria** [comment]
@tobilen so im not sure what's different with the recent compile but it seems to be murdering my fps by at least 20 frames, it went from 98  to 68-78ish and noticeable stuttering, i uninstalled it and it reverted to normal

**Comment by TyDodson** [comment]
> @tobilen so im not sure what's different with the recent compile but it seems to be murdering my fps by at least 20 frames, it went from 98 to 68-78ish and noticeable stuttering, i uninstalled it and it reverted to normal

I am also experiencing this issue with the latest build of the Endwalker update branch of @tobilen fork. The FPS seem to slow down as soon as the icons appear on screen and go back to normal after about 30 seconds.

**Comment by SeleneBathory** [comment]
Hi,

Same problem with fps, I'm loosing 20fps when icon appear (test on DNS). Any chance to have an update ? It's really a great addon.
Thanks.

**Comment by ProfessorBAG** [comment]
Does this plugin not work anymore?
